README.md

```diff
@@ -41,6 +41,7 @@ PLATFORM=Darwin
 ARCH=arm64
 VERSION=0.0.1
```

Comment:
Can we update this to the hack version. And save figuring out a dynamic solution for future work ?

Reply:
done